### DISCUSSION TITLE
这频道不会对github产生额外占用吧？

### DISCUSSION THREAD

**Discussion by Xy9223** [share121/inter-knot#3169] [其他]
是怎么实现的？很好奇

**Comment by x97-1** [comment]
放轻松，光在github代理的网站少说都有几百万个了，就算cloudflare，face崩了github可能都还坚挺，当然这三个崩的可能性都极低。

> 与其担心这些还不如去多喝杯奶茶

**Reply by cpu644** [reply]
之前不有个13Tbps的ddos被拿来打git了吗 十几分钟拿下  甚至都没下线 确实不如去喝奶茶。
